src/API/Site/Controllers/Jetpack/AccountController.php

```diff
@@ -30,6 +31,18 @@ class AccountController extends BaseOptionsController {
 	 */
 	protected $middleware;
 
+	/**
+	 * Mapping between the client page name and its path.
+	 * The first value is also used as a default,
+	 * and changing the order of keys/values may affect things below.
+	 *
+	 * @var string[]
+	 */
+	private const NEXT_PATH_MAPPING = [
```

Comment:
💅  About `next` naming in general 

Was not clear at first glance what that `next` param was when I looked at the frontend. I needed to read the docs and the code to finally understood that was meant the URL Callback provided by the Authentication Jetpack API.

WDYT about `url_callback`? Seems like its how they named those in [Auth0](https://auth0.github.io/auth0-oidc-client-net/documentation/getting-started/callbacks.html)

Reply:
Same as https://github.com/woocommerce/google-listings-and-ads/pull/1385#discussion_r840314974.